tests/e2e/lib.sh

```diff
@@ -94,6 +94,8 @@ export_test_environment() {
     ci_export LOCAL_PORT "${LOCAL_PORT:-443}"
     ci_export MONITORING_SUPPORT "${MONITORING_SUPPORT:-false}"
     ci_export SCANNER_SUPPORT "${SCANNER_SUPPORT:-true}"
+    ci_export USE_MIDSTREAM_IMAGES "${USE_MIDSTREAM_IMAGES:-false}"
+    ci_export REMOTE_CLUSTER_ARCH "${REMOTE_CLUSTER_ARCH:-amd64}"
```

Comment:
@jdao-rh  we need to set `REMOTE_CLUSTER_ARCH` to `x86_64` since that's the default value used in test suite https://github.com/stackrox/stackrox/blob/master/qa-tests-backend/src/main/groovy/util/Env.groovy#L31C81-L31C87